### PR TITLE
DAOS-4757-test: remove skipForTicket rebuild read_array.py

### PR DESCRIPTION
**Pull request by dinghwah**
modified:   read_array.py
modified:   read_array.yaml
Skip-unit-tests: true
Skip-nlt: true
Quick-Functional: true
Test-tag: rebuildreadarray
Signed-off-by: Ding Ho ding-hwa.ho@intel.com

### CODE REVIEW

**Comment by dinghwah** [comment]
All tests on the Test-tag are passed.

File: src/tests/ftest/rebuild/cascading_failures.py [hunk 1]
```diff
@@ -14,7 +14,7 @@ class CascadingFailures(RebuildTestBase):
     :avocado: recursive
     """
 
-    CANCEL_FOR_TICKET = [["DAOS-2799", "targets", 8]]
+    CANCEL_FOR_TICKET = [["DAOS-6728"]]
```
Comment (saurabhtandan) [inline src/tests/ftest/rebuild/cascading_failures.py:17]: Don't think this is the correct way of using `CANCEL_FOR_TICKET` option. The ticket id should be accompanied with some condition from yaml file or the test case.

**Review by saurabhtandan — CHANGES_REQUESTED** [review]
Also, can you please update the tags of cascading_failures.py to have common tag for the whole test as well. And run this PR with that tag even if other tests are supposed to be skipped. I think because the PR was run with just 'cascading' tag, the issue of `CANCEL_FOR_TICKET` mentioned below might not have been picked up.

==>Done.

**Review by shimizukko — APPROVED** [review]

**Comment by dinghwah** [comment]
All tests passed from the testcases on the test-Tag.

**Review by ravalsam — APPROVED** [review]

**Review by saurabhtandan — CHANGES_REQUESTED** [review]
Results of Build 3 of this pr seem to indicate 2 cascading tests actually passed. But later all the tests are being skipped. Can we unsnap that test please.

**Pull request merged**